src/libs/actions/ReimbursementAccount/setupWithdrawalAccount.js

```diff
@@ -241,12 +236,12 @@ function setupWithdrawalAccount(params) {
             } else {
                 navigation.goToWithdrawalAccountSetupStep(nextStep, responseACHData);
             }
-            Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {loading: false});
+            Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {isLoading: false});
         })
         .catch((response) => {
-            Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {loading: false, achData: {...updatedACHData}});
+            Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {isLoading: false, achData: {...updatedACHData}});
             console.error(response.stack);
-            errors.showBankAccountErrorModal(Localize.translateLocal('common.genericErrorMessage'));
+            errors.showBankAccountFormValidationError('common.genericErrorMessage');
```

Comment:
We still need to translate this error key, right? Otherwise we will just display `'common.genericErrorMessage'` to the user